routes/videos/VideosController.cs

```diff
@@ -262,4 +262,116 @@ public async Task<IActionResult> AddVideoComment(YoutubeContext db, long videoId
 
         return Ok();
     }
+     /* [HttpGet("{videoId}/like")] //do details albo request
+ [Authorize(AuthenticationSchemes = CookieAuthenticationDefaults.AuthenticationScheme)]
+ [RequiresUserAccount]
+ public async Task<IActionResult> GetVideoLikes(YoutubeContext db, int videoID )
+ {
+     var userId = User.FindFirst(ClaimTypes.NameIdentifier)?.Value;
+
+     
+     var userRating = await db.Likes
+         .Where(l => l.Video == videoID && l.Account == long.Parse(userId))
+         .Select(l => new { l.Unlike })
+         .FirstOrDefaultAsync();
+
+     if (userRating == null)
+     {
+         return Ok(new { rated = false });
+     }
+
+     return Ok(new
+     {
+         rated = true,
+         dislike = userRating.Unlike // 'true' jeśli to dislike, 'false' jeśli to like
+     });
+ }
+ */
+
+     [HttpPost("{videoId}/like")]
+    [Authorize(AuthenticationSchemes = CookieAuthenticationDefaults.AuthenticationScheme)]
+    [RequiresUserAccount]
+    public async Task<IActionResult> LikeVideo(YoutubeContext db, [FromBody] AddVideoCommentModel body)
+    {
+        Video? video = await db.Videos.FindAsync((int)body.videoId);
+        if (video == null)
+        {
+            return NotFound("404");
+        }
+
+        var userId = User.FindFirst(ClaimTypes.NameIdentifier);
+
+        Like? likePrevID = await db.Likes
```

Comment:
var likePrevID

Reply:
Znaczy to też i tak będzie do usunięcia, ponieważ nie powinno być wyliczane id kolejnego elementu tutaj